manageiq-operator/pkg/helpers/miq-components/cr.go

```diff
@@ -97,6 +97,22 @@ func httpdImageTag(cr *miqv1alpha1.ManageIQ) string {
 	}
 }
 
+func httpdImage(cr *miqv1alpha1.ManageIQ, privileged bool) string {
+	if cr.Spec.HttpdImage == "" {
+		var image string
+
+		if privileged {
+			image = "httpd-init"
+		} else {
+			image = "httpd"
+		}
```

Comment:
Would this be simpler?
```suggestion
		if privileged {
			image := "httpd-init"
		} else {
			image := "httpd"
		}
```

Reply:
Actually that won't work,  undefined: image because the image := are only defined within the scope of the if blocks.  we need the above var declaration. I miss Ruby.

Reply:
Oh, right. I miss Ruby too.